### PR TITLE
feature/show-bot-path

### PR DESCRIPTION
**Pull request by edsanville**

### CODE REVIEW

File: src/web/jcc/client/components/CommandControl.tsx [hunk 1]
```diff
@@ -801,6 +807,70 @@ export default class CommandControl extends React.Component {
 		}
 	}
 
+	
+	/**
+	 * Use JaiaAPI to poll the botPath data.
+	 */
+	pollBotPaths() {
+		console.log(this.lastBotPathPointUtime)
```
Comment (marshalllele) [inline src/web/jcc/client/components/CommandControl.tsx]: This console.log()  gunks up the console a good bit

**Review by marshalllele — CHANGES_REQUESTED** [review]

File: src/web/jcc/client/components/CommandControl.tsx (hunk 1 above)
Comment (edsanville) [reply src/web/jcc/client/components/CommandControl.tsx]: Removed!

File: src/web/jcc/client/components/CommandControl.tsx [hunk 2]
```diff
@@ -74,6 +74,7 @@ import '../style/CommandControl.less'
 // Utility
 import cloneDeep from 'lodash.clonedeep'
 import { HelpWindow } from './HelpWindow'
+import { getBotPathColor } from './shared/BotPathColors'
```
Comment (marshalllele) [inline src/web/jcc/client/components/CommandControl.tsx]: It looks like this is an unused import

**Review by marshalllele — CHANGES_REQUESTED** [review]

**Review by marshalllele — APPROVED** [review]
Looks good! Tested both paths in sim and in JDV.

**Pull request merged**